### DISCUSSION TITLE
Can Thread device discover other Thread/IP based devices?

### DISCUSSION THREAD

**Discussion by lecontr2** [openthread/openthread#8489] [Q&A] [answered]
I was wondering if this capability is built into OpenThread? I'm thinking of a scenario where a new Thread device that joins a network is able to find other Thread devices and request information from them. 

Additionally, as I understand it, a Thread node can use the Border Router as a Discovery Proxy for mDNS but is that feature mandatory for certification? Can a border router be used for SSDP to discover devices on the Wi-Fi/Ethernet side as well?

**Comment by jwhui** [comment, marked as answer]
Thread 1.3 includes support for DNS-Based Service Discovery. This allows Thread devices to discover services using standard unicast DNS and Wi-Fi / Ethernet devices to discover services using mDNS.

Thread 1.3 devices are not required to use DNS-Based Service Discovery to publish or discover services. However, Thread 1.3 Border Routers are required to support SRP Server, DNS-SD Advertising Proxy, and Discovery Proxy.

The [Thread Border Router codelab](https://openthread.io/codelabs/openthread-border-router) provides an example of how to use these services.

**Reply by ksathara** [reply]
Hi, I am looking into OpenThread and Thread 1.0 interoperability. We have used Thread 1.0 previously on many devices and want to know if we can communicate with them using our new OpenThread devices. How can this be done?
Cheers,
Kaushalya
